testdata/go/v3/monitoring/memcached-operator/README.md

```diff
@@ -1,8 +1,8 @@
-# memcached-operator
-// TODO(user): Add simple overview of use/purpose
+# memcached-operator (with monitoring)
+A Go-based Operator framework that has the goal to install, deploy, monitor and manage [Memcached](https://hub.docker.com/_/memcached) on the cluster.
 
 ## Description
-// TODO(user): An in-depth paragraph about your project and overview of use
+This project is based on the testdata operator and will deploy the operator with the recommended structure and tooling for metrics, alerts, runbooks and all that is needed for monitoring with Prometheus, in order to help developers start implementing monitoring and provide them with tooling, examples and best practices.
```

Comment:
I didn't see anywhere in the generation logic that this was added. Did you manually add this information? If you manually added this information the sanity check will fail due to a diff in the contents when running `make generate` (which will overwrite these changes if they were made manually). Instead of manually adding this information, the generation logic would have to be updated to incorporate these changes.

That being said, I'm not sure there is much value in having this information present in the testdata README unless it is completely undocumented elsewhere. It does appear that this is documented here: https://sdk.operatorframework.io/docs/building-operators/golang/advanced-topics/#monitoring-and-observability

Reply:
Hi @everettraven I thought it might be generated buy I didn't found the generation file. I searched in github some strings that were in this file but no results. can you share the generation file location?

Reply:
But if you feel it's not needed at all, I can just remove all changes to the README. please let me know.